src/main/java/org/cryptomator/cryptofs/health/dirid/EmptyDirFile.java

```diff
@@ -0,0 +1,49 @@
+package org.cryptomator.cryptofs.health.dirid;
+
+import org.cryptomator.cryptofs.health.api.DiagnosticResult;
+
+import java.nio.file.Path;
+import java.util.Map;
+
+import static org.cryptomator.cryptofs.health.api.CommonDetailKeys.DIR_ID_FILE;
+
+/**
+ * A critical diagnostic result of an empty directory id file (dir.c9r).
```

Comment:
It is only "critical" (i.e. no fix available), because there is no fix implemented _yet_, right?

Reply:
Yes. I planned to propose this change to all DiagnosticResult classes as long as no fix is implemented for those.

Reply:
Fair enough, but I'm afraid the JavaDoc won't be adjusted too often and shouldn't reflect the work-in-progress but rather the intended role of this class.

Reply:
Fixed in 0df4948